commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/CPDefinitionsImporter.java

```diff
@@ -189,7 +191,8 @@ private CPDefinition _addCPDefinition(
 			long catalogGroupId, String name, String shortDescription,
 			String description, String externalReferenceCode, String sku,
 			String taxCategory, long width, long height, long depth,
-			long weight, long[] assetCategoryIds, String[] assetTagNames,
+			long weight, JSONObject subscriptionInfoJSONObject,
```

Comment:
@Commando18 Please avoid to pass JSONObject, better pass all fields needed

Reply:
Fixed, thanks!